elements/pl-checkbox/pl-checkbox.py

```diff
@@ -365,23 +378,26 @@ def parse(element_html, data):
     element = lxml.html.fragment_fromstring(element_html)
     name = pl.get_string_attrib(element, 'answers-name')
 
+    allow_blank = pl.get_boolean_attrib(element, 'allow-blank', ALLOW_BLANK_DEFAULT)
+
     submitted_key = data['submitted_answers'].get(name, None)
     all_keys = [a['key'] for a in data['params'][name]]
 
-    # Check that at least one option was selected
-    if submitted_key is None:
-        data['format_errors'][name] = 'You must select at least one option.'
-        return
-
-    # Check that the selected options are a subset of the valid options
-    # FIXME: raise ValueError instead of treating as parse error?
-    submitted_key_set = set(submitted_key)
-    all_keys_set = set(all_keys)
-    if not submitted_key_set.issubset(all_keys_set):
-        one_bad_key = submitted_key_set.difference(all_keys_set).pop()
-        # FIXME: escape one_bad_key
-        data['format_errors'][name] = 'You selected an invalid option: {:s}'.format(str(one_bad_key))
-        return
+    if allow_blank == False:
```

Comment:
options here are `if allow_blank is False` (if its important to distinguish between False and None). 

If not important (usual case), the I think `if not allow_blank:`